### PR TITLE
feat(homepage): frontend, not connected to api

### LINKED ISSUE #24: Create a game with a custom game ID and 10 rounds

### PR DESCRIPTION
**Pull request by hellasol**
resolves #24 
part of issue 9 on the server

### CODE REVIEW

File: src/components/views/CreateGame.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React, {useState} from 'react';
 import {useHistory} from 'react-router-dom';
 import BaseContainer from "components/ui/BaseContainer";
+import {api, handleError} from 'helpers/api';
+import Lobby from "models/Lobby";
```
Comment (sumsumcity) [inline src/components/views/CreateGame.js:5]: I think should be: 

import Lobby from "./Lobby";

I get an error with "models/Lobby"

File: src/components/views/HomePage.js [hunk 2]
```diff
@@ -2,9 +2,76 @@ import React from 'react';
 import {useHistory} from 'react-router-dom';
 import BaseContainer from "components/ui/BaseContainer";
 import 'styles/views/HomePage.scss';
+import { useEffect } from 'react';
+import Lobby from 'models/Lobby';
```
Comment (sumsumcity) [inline src/components/views/HomePage.js:6]: I think should be: 

import Lobby from "./Lobby";

I get an error with "models/Lobby"

Comment (hellasol) [reply src/components/views/HomePage.js]: thanks 👍

**Comment by sumsumcity** [comment]
I found some things but maybe some of those aren't part of this Pull Request:

- [ ] No creation of a lobby after creating the game. (Lobby issue (Perhaps not in this user story?))
- [ ] Always 2/4 waiting in the lobby. Even if there should be 10 players (Lobby issue)
- [ ] Game status change (Lobby issue)
- [ ] All other pages have a different styling than before. So those doesnt look good afterwards. E.g. userlist or scoreboard
- [ ] No list of games at the home screen

**Comment by radubauzh** [comment]
@hellasol @sumsumcity 

I can confirm all the issues Raphael has stated above, this branch still some work. The styling in this branch somehow changed the whole appearance (see below)

main
This branch

**Comment by hellasol** [comment]
> * [ ]  No creation of a lobby after creating the game. (Lobby issue (Perhaps not in this user story?)) -> should be addressed in #17, not relevant here
> * [ ]  Always 2/4 waiting in the lobby. Even if there should be 10 players (Lobby issue) -> should be adressed in #17, not relevant here 
> * [ ]  Game status change (Lobby issue) -> should be adressed in #17, not relevant here 
> * [ ]  All other pages have a different styling than before. So those doesnt look good afterwards. E.g. userlist or scoreboard -> we were using the same css classes.. I changes mine. We should start prefixing the css classes to avoid this. 
> * [ ]  No list of games at the home screen -> this should be addressed in #4 'join avalable game', not relevant here 
> * [ ]  Routing -> what do you mean? in general? we could open a sperate ticket for that. 

@sumsumcity please try to keep within the scope of the ticket 👍

**Review by sumsumcity — COMMENTED** [review]
Okay thanks then i think it is good to go @hellasol

**Pull request merged**